drivers/ieee802154/ieee802154_nrf5.c

```diff
@@ -594,7 +594,7 @@ static uint8_t nrf5_get_acc(const struct device *dev)
 {
 	ARG_UNUSED(dev);
 
-	return CONFIG_IEEE802154_DELAY_TRX_ACC;
+	return CONFIG_CLOCK_CONTROL_NRF_ACCURACY;
```

Comment:
I wonder if in some cases it will be desirable to establish a different value for CSL Accuracy than the main clock accuracy. Users might decide to risk a bit on missing CSL receptions on favor of longer battery life, instead of accounting for worst case drifts.

I would keep `CONFIG_IEEE802154_DELAY_TRX_ACC` option but assign it `CONFIG_CLOCK_CONTROL_NRF_ACCURACY` as default value.